src/Tools/tools.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <SerializeProjects>true</SerializeProjects>
```

Comment:
Nevermind I was only looking at the one commit and I now see you have 2 projects that you do need to serialize. Although I really dislike having projects depend on each other in this way :(, I can live with it though since it is just temporary.